test/per_file_coverage.sh

```diff
@@ -58,9 +58,9 @@ declare -a KNOWN_LOW_COVERAGE=(
 "source/extensions/rate_limit_descriptors/expr:95.5"
 "source/extensions/stat_sinks/graphite_statsd:82.1"
 "source/extensions/stat_sinks/statsd:84.6"
-"source/extensions/tracers:95.8"
-"source/extensions/tracers/common:73.8"
-"source/extensions/tracers/common/ot:71.7"
+"source/extensions/tracers:95.5"
+"source/extensions/tracers/common:68.1"
+"source/extensions/tracers/common/ot:65.6"
```

Comment:
>i added a senior maintainer pass mainly for the coverage override. A shared /ot component was relying on Datadog for testing coverage, and Datadog is no longer user/testing that shared component; the owner of the 'ot' component should own the fixing of its test coverage.

If this PR moves datadog off of shared code and means the shared code is no longer tested, I think adding back some tests is within scope of the PR.

Reply:
@alyssawilk precedence says otherwise: https://github.com/envoyproxy/envoy/commit/eb521f42e760b9e5cb6ca544e5c87523f0592fa9#diff-0ac04ef31c616358d8234bd473469c1a233ecf37947a13475cce764e7f6c43eeR79-R81

Reply:
That was a removal which blocked key dependency upgrades and we were willing to make allowances.  I don't think such an exception applies here.